### ISSUE TITLE
Prettier graphs

### ISSUE THREAD

**Issue by lexicalunit** [theastropath/turbot#16]
Having all the users and all their prices on a single graph is a bit hard to parse and just looks like really colorful spaghetti.

It might be more useful to try and get something like this working so you could see a gallery of users' graphs: https://python-graph-gallery.com/125-small-multiples-for-line-chart/

Also if we could do whatever https://turnipprophet.io/ is doing that would be really amazing.

**Comment by lexicalunit** [comment]
Now that we have the `!predict` command we might not really need the `!turnippattern` command anymore. The graphs are still cool to have tho.

**Comment by theastropath** [comment]
We could leave !turnippattern there for quick reference, but maybe add a line that says something like "Try the !predict command for more information!"

**Comment by lexicalunit** [comment]
We might be able to leverage this library for pretty graphs **and** predictions: https://gitlab.com/nanoNago/turnips

![toms_island](https://user-images.githubusercontent.com/1903876/80844467-72d61780-8bbb-11ea-88c8-ef1869e425a9.png)
